src/components/standalone/users_objects/CreateOrEditHostSetDrawer.vue

```diff
@@ -105,7 +106,18 @@ watch(
   }
 )
 
+// compute redirectItemsName the name of the portforward rule using this object
+watch(
+  () => props.currentHostSet?.matches,
+  async () => {
+    if (props.currentHostSet?.matches) {
+      redirectItemsName.value = await getMatchedItemsName(props.currentHostSet.matches)
+    }
```

Comment:
```suggestion
  async (matches) => {
    if (matches) {
      redirectItemsName.value = await getMatchedItemsName(matches)
    }
```

Reply:
sorry but vscode is complaining :p

![image](https://github.com/user-attachments/assets/e4729960-2148-4dfe-b676-1bd99c38fee4)

Reply:
You're missing the `async (matches)` above! :grimacing:

Reply:
better, thank